From f68ed967aff06ce80a7fe6c08677b5bc4c01a1a4 Mon Sep 17 00:00:00 2001
From: Harry Youd <harry@harryyoud.co.uk>
Date: Sun, 9 Jul 2017 10:20:18 +0100
Subject: [PATCH 100/249] Various fixes to README

- Use https:// links where ever possible
- Fix some grammar and punctuation
- Remove reference to non-existent wiki page
- Mark commands as inline code
- Add link to downloads

Change-Id: I4620030b91c887dc41bfd02e73fcdbf71fd88452
---
 README.mkdn | 26 ++++++++++++++------------
 1 file changed, 14 insertions(+), 12 deletions(-)

diff --git a/README.mkdn b/README.mkdn
index 37742ae..8538697 100644
--- a/README.mkdn
+++ b/README.mkdn
@@ -3,7 +3,8 @@ LineageOS
 
 Submitting Patches
 ------------------
-Patches are always welcome!  Please submit your patches via LineageOS Gerrit!
+
+Patches are always welcome!  Please submit your patches via LineageOS Gerrit.
 You can do this by using these commands:
 
     (From root android directory)
@@ -14,20 +15,21 @@ You can do this by using these commands:
     repo start cm-14.1 .
     (Make your changes and commit)
     repo upload .
-Note: "." meaning current directory
-For more help on using this tool, use this command: repo help upload
 
-Make your changes and commit with a detailed message, starting with what you are working with (i.e. vision: Update Kernel)
-Commit your patches in a single commit. Squash multiple commit using this command: git rebase -i HEAD~<# of commits>
+Note: "." means the current directory
+For more help on using this tool, use this command: `repo help upload`
+
+Make your changes and commit with a detailed message, starting with what you are working with (i.e. "vision: Update Kernel")
+Commit your patches in a single commit. Squash multiple commits using this command: `git rebase -i HEAD~<# of commits>`
 
-To view the status of your and others' patches, visit [LineageOS Code Review](http://review.lineageos.org/)
+To view the status of your and others' patches, visit [LineageOS Code Review](https://review.lineageos.org/)
 
 
 Getting Started
 ---------------
 
 To get started with Android/LineageOS, you'll need to get
-familiar with [Git and Repo](http://source.android.com/source/using-repo.html).
+familiar with [Git and Repo](https://source.android.com/source/using-repo.html).
 
 To initialize your local repository using the LineageOS trees, use a command like this:
 
@@ -37,14 +39,14 @@ Then to sync up:
 
     repo sync
 
-Please see the [LineageOS Wiki](http://wiki.lineageos.org/) for building instructions.
+Please see the [LineageOS Wiki](https://wiki.lineageos.org/) for building instructions.
 
-For more information on this Github Organization and how it is structured,
-please [read the wiki article](http://wiki.lineageos.org/w/Github_Organization)
 
 Buildbot
 --------
 
-All supported devices are built nightly and periodically as changes are committed to ensure the source trees remain buildable.
+All supported devices are built weekly and periodically as changes are committed to ensure the source trees remain buildable.
+
+You can view the current build statuses at [LineageOS Jenkins](https://jenkins.lineageos.org/)
 
-You can view the current build statuses at [LineageOS Jenkins](http://jenkins.lineageos.org/)
+Builds produced weekly by the buildbot can be downloaded from [LineageOS downloads](https://download.lineageos.org/)
-- 
2.17.1

